pkg/events/log_events.go

```diff
@@ -37,17 +45,31 @@ func (l loggingEventHandler) Handle(u *unstructured.Unstructured, e eventapi.Job
 		"gvk":        u.GroupVersionKind().String(),
 		"event_type": e.Event,
 	})
-	t, ok := e.EventData["task"]
-	if ok {
-		log = log.WithField("task", t)
+
+	if l.LogLevel == Nothing {
+		return
+	}
+
+	_, ok := e.EventData["task"]
+
+	// log only the following for the 'Tasks' LogLevel
+	if ok &&
+	   e.Event == EventPlaybookOnTaskStart &&
+	   e.EventData["task_action"] != TaskActionDebug &&
```

Comment:
I would recommend making `task_action` a `const`. You use it in multiple places.

Reply:
You could also make a local var for task action rather than accessing the map every time. This is just a nit though